src/types.ts

```diff
@@ -0,0 +1,309 @@
+/*
```

Comment:
For folks using typescript, wouldn't they need to import these types with a suffix? Like this:

```typescript
import { InvocationEvent, Context, Logger, RecordQueryResult } from 'sf-fx-runtime-nodejs/types';
```

Would it make more sense to name this file `index.ts` instead, so they could be imported from `'sf-fx-runtime-nodejs'` (without a suffix)?

Reply:
Or if we don't move the file, we could at least re-export from `index.ts`?